### PR TITLE
feat: fix documentation comments for methods used to mount a bucket in Alluxio

### PR DESCRIPTION
**Pull request by abnerferreira0212**
# Pull Request Merge Checklist
           
Task backlog identifier related to this PR: 1542

---

## As a DEVELOPER, I confirm that this PR **doesn't contain**:
- [x] Code lines with syntax errors
- [x] Methods and variables names that aren't clear or doesn't mean anything
- [x] Code that is out of the standard (Example: **PEP-8** for Python code)
- [x] Files with **imports** errors or without any utility
- [x] **TODOs** left behind
- [x] Tests that are being skipped
- [x] Methods placed on incorrect files or folders (Example: methods that are local, but should be placed on /utils)
- [x] Functions, methods or API responses that are publicly returning sensitive information

## As a DEVELOPER, I confirm that I:
- [x] Followed the Simbiose's standard for branch and commit naming
- [ ] Concluded the task and achieved the goals according to the task planning I defined before starting to code
- [ ] Added all necessary database changes to the [SQL Migrations](https://github.com/s1mbi0se/sql-migrations)
- [ ] Created tests for every new flow and method on this Pull Request
- [ ] Added the necessary DEBUG logs
- [x] Checked my code using the Simbiose's Corner Cases Guide
- [x] Merged the **develop** branch to this PR/branch before sending the PR for review
- [ ] Solved all possible merge conflicts before sending the PR for review
- [ ] Corrected all the problems and code smells identified by the quality tools
- [ ] Manually executed the necessary tests locally to make sure my changes didn't break anything
- [ ] All unitary and integration tests ran by the CI system are passing
- [ ] Check the performance of my changes or additions and they didn't make the system slower (I profiled the code)
- [x] Updated the necessary documentation (Example: OpenAPI, Docstrings, Metadata Database Modeling)
- [ ] Made the self-review of the code before sending the PR for review
- [ ] Sent this PR for a peer-review
- [x] Made all the corrections appointed during the peer-review
- [x] Confirmed that this PR merge will not cause other systems or dependencies to stop working (Example: ShannonDB Python Client and ShannonDB)
- [ ] Checked all GitHub Actions related to this PR that can block my merge

---  

## As the TEAM MANAGER, I confirm that I:
- [x] Checked that the code was developed following the best practices for the language, with the best possible readability and using the less possible quantity of code lines and complexity. 
- [x] Checked if the Pull Request and commit messages follow the Simbiose’s defined standard 
- [x] Checked if there is no possible merge conflicts before reviewing the PR
- [x] Checked if the purpose of the task was reached with this PR
- [x] Checked that the code was developed following the best practices for the language, with the best possible readability and using the less possible quantity of code lines and complexity. 
- [x] Checked if all related PRs were created when other components were affected by this task (e.g. database changes on [SQL Migrations](https://github.com/s1mbi0se/sql-migrations))
- [x] Checked that were created unit, integration and chaos tests for every new flow and method on this Pull Request
- [x] Checked that no sensible information (e.g. users security data, internal security data, etc) was exposed by this PR
- [x] Checked that code coverage for this PR is higher than 80%.
- [x] Checked that there are no problems and code smells identified by the quality and security tools
- [x] Checked that all files were created on the proper folders according to the Simbiose’s defined standard
- [x] Checked that all classes, methods and variables on this PR follow the Simbiose’s pattern for this programming language
- [x] Checked that all unitary and integration tests ran by the CI system are passing
- [x] Checked that there are no performance issues created by these changes or additions
- [x] Checked that all documentation is up to date according to this change (Example: OpenAPI, Docstrings, Metadata Database Modeling)
- [x] Checked that this PR merge will not cause other systems or dependencies to stop working (Example: ShannonDB Python Client and ShannonDB)
- [x] Checked all GitHub Actions related to this PR that can block the merge

---  

## As the QUALITY RESPONSIBLE, I confirm that I:
- [x] Checked if the Pull Request and commit messages follow the Simbiose’s defined standard 
- [x] Checked if there is no possible merge conflicts before reviewing the PR
- [x] Checked that were created unit, integration and chaos tests for every new flow and method on this Pull Request
- [x] Checked that no sensible information (e.g. users security data, internal security data, etc) was exposed by this PR
- [ ] Checked that code coverage for this PR is higher than 80%.
- [ ] Checked that there are no problems and code smells identified by the quality and security tools
- [x] Checked that all files were created on the proper folders according to the Simbiose’s defined standard
- [x] Checked that all classes, methods and variables on this PR follow the Simbiose’s code standard for this programming language
- [ ] Checked that all unitary and integration tests ran by the CI system are passing
- [ ] Checked that there are no performance issues created by these changes or additions
- [x] Checked that all documentation is up to date according to this change (Example: OpenAPI, Docstrings, Metadata Database Modeling)
- [x] Checked that this PR merge will not cause other systems or dependencies to stop working (Example: ShannonDB Python Client and ShannonDB)
- [x] Checked all GitHub Actions related to this PR that can block the merge

## Important links:
1. [Simbiose Quality Guidelines](https://github.com/s1mbi0se/company/wiki/Quality-Topics#table-of-contents)
2. [Metadata Database Modeling](https://docs.google.com/spreadsheets/d/1Y8_k5xV86kTjgfjbi-PGsJcubUKhWlT2x8ecpnMlyR0/edit?usp=sharing)
3. [Simbiose's Corner Cases Guide](https://www.simbioseventures.com/documentos-e-guias/procedimentos-e-checklists/corner-cases-guide)

### CODE REVIEW

File: core/common/src/main/java/alluxio/security/login/AlluxioLoginModule.java [hunk 1]
```diff
@@ -86,11 +66,12 @@ public boolean abort() throws LoginException {
 
   /**
    * Commits the authentication (second phase).
-   *
+   * <p>
    * This method is called if the LoginContext's overall authentication succeeded. (login
    * succeeded)
```
Comment (felipe-dna) [inline core/common/src/main/java/alluxio/security/login/AlluxioLoginModule.java:71]: The final point must be after the (), what you think?

File: core/common/src/main/java/alluxio/security/user/UserStateFactory.java [hunk 2]
```diff
@@ -20,6 +20,8 @@
  */
 public interface UserStateFactory {
   /**
+   * Returns a new user state
```
Comment (felipe-dna) [inline core/common/src/main/java/alluxio/security/user/UserStateFactory.java:23]: Missing final point

File: core/common/src/main/java/alluxio/util/CommonUtils.java [hunk 3]
```diff
@@ -769,8 +769,6 @@ public static String convertMsToClockTime(long millis) {
    * Updates and returns supplier, implementing memoization
```
Comment (felipe-dna) [inline core/common/src/main/java/alluxio/util/CommonUtils.java:769]: Missing final point

File: underfs/b2/src/main/java/alluxio/underfs/b2/B2InputStream.java [hunk 4]
```diff
@@ -0,0 +1,147 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
```
Comment (felipe-dna) [inline underfs/b2/src/main/java/alluxio/underfs/b2/B2InputStream.java:4]: Missing final point

File: underfs/b2/src/main/java/alluxio/underfs/b2/B2UnderFileSystem.java [hunk 5]
```diff
@@ -0,0 +1,286 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
```
Comment (felipe-dna) [inline underfs/b2/src/main/java/alluxio/underfs/b2/B2UnderFileSystem.java:4]: Missing final point.

File: underfs/b2/src/main/java/alluxio/underfs/b2/B2UnderFileSystemFactory.java [hunk 6]
```diff
@@ -0,0 +1,61 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
```
Comment (felipe-dna) [inline underfs/b2/src/main/java/alluxio/underfs/b2/B2UnderFileSystemFactory.java:4]: Missing final point.

File: underfs/b2/src/main/java/alluxio/underfs/b2/B2Utils.java [hunk 7]
```diff
@@ -0,0 +1,49 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
```
Comment (felipe-dna) [inline underfs/b2/src/main/java/alluxio/underfs/b2/B2Utils.java:4]: Missing final point

File: underfs/b2/src/main/resources/META-INF/services/alluxio.underfs.UnderFileSystemFactory [hunk 8]
```diff
@@ -0,0 +1,12 @@
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the “License”). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
```
Comment (felipe-dna) [inline underfs/b2/src/main/resources/META-INF/services/alluxio.underfs.UnderFileSystemFactory:4]: Missing final point

File: underfs/b2/src/test/java/alluxio/underfs/b2/B2InputStreamTest.java [hunk 9]
```diff
@@ -0,0 +1,103 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
```
Comment (felipe-dna) [inline underfs/b2/src/test/java/alluxio/underfs/b2/B2InputStreamTest.java:4]: Missing final point

File: underfs/b2/src/test/java/alluxio/underfs/b2/B2UnderFileSystemFactoryTest.java [hunk 10]
```diff
@@ -0,0 +1,37 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
```
Comment (felipe-dna) [inline underfs/b2/src/test/java/alluxio/underfs/b2/B2UnderFileSystemFactoryTest.java:4]: Missing final point.

File: underfs/b2/src/test/java/alluxio/underfs/b2/B2UtilsTest.java [hunk 11]
```diff
@@ -0,0 +1,56 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
```
Comment (felipe-dna) [inline underfs/b2/src/test/java/alluxio/underfs/b2/B2UtilsTest.java:4]: Missing final  point.

File: shell/src/main/java/alluxio/cli/fs/command/UnmountCommand.java [hunk 12]
```diff
@@ -55,6 +55,21 @@ protected void runPlainPath(AlluxioURI inputPath, CommandLine cl)
...
+   * Keeps all command line arguments stored in a collection.
+   * The first argument should always correspond to the {@link AlluxioURI}
+   * of the Alluxio path to be deleted.
+   * <p>
+   * Returns the exit code 0 if the operation is completed successfully.
+   * Throws an exception otherwise.
+   *
+   * @param   cl  the parsed command line for the arguments
+   * @return  the exit code 0 (zero) if the command runs successfully
+   * @throws  IOException       if the provided URI does not exist or if
```
Comment (felipe-dna) [inline shell/src/main/java/alluxio/cli/fs/command/UnmountCommand.java:70]: Throws must start with uppercase. Missing final point too.

File: core/server/master/src/main/java/alluxio/master/file/FileSystemMaster.java [hunk 13]
```diff
@@ -414,10 +414,10 @@ void mount(AlluxioURI alluxioPath, AlluxioURI ufsPath, MountContext context)
...
    * of the Alluxio path.
    *
-   * @param alluxioPath the Alluxio path to unmount, must be a mount point
-   * @throws FileDoesNotExistException if the path to be mounted does not exist
-   * @throws InvalidPathException if the given path is not a mount point
-   * @throws AccessControlException if the permission check fails
+   * @param   alluxioPath the Alluxio path to unmount, must be a mount point
+   * @throws  FileDoesNotExistException  if the path to be mounted does not exist
+   * @throws  InvalidPathException       if the given path is not a mount point
+   * @throws  AccessControlException     if the permission check fails
```
Comment (felipe-dna) [inline core/server/master/src/main/java/alluxio/master/file/FileSystemMaster.java:420]: Throws must start with uppercase. Missing final point too.

File: core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java [hunk 14]
```diff
@@ -2830,14 +2850,33 @@ public void unmount(AlluxioURI alluxioPath) throws FileDoesNotExistException,
...
    * inode containing deleted blocks.
    *
-   * @param rpcContext the rpc context
-   * @param inodePath the Alluxio path to unmount, must be a mount point
+   * @param rpcContext  the {@link RpcContext} provided by {@link #unmount(AlluxioURI)}
+   * @param inodePath   the Alluxio path to unmount, must be a mount point
+   * @throws FileDoesNotExistException  if the full {@code inodePath} does not exist
+   * @throws InvalidPathException       if information about a mount point does not
+   *                                    exist or the path to be deleted is root
+   * @throws RuntimeException           if the operation fails because the directory is not empty.
```
Comment (felipe-dna) [inline core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java:2877]: Throws must start with uppercase. Missing final point too.

File: core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java [hunk 15]
```diff
@@ -2806,6 +2806,26 @@ private void mountInternal(Supplier<JournalContext> journalContext, LockedInodeP
...
+   * <p>
+   * Executes {@link #unmountInternal(RpcContext, LockedInodePath)} with the created {@code inodePath}
+   * and {@code rpcContext}.
+   *
+   * @param alluxioPath the Alluxio path to unmount, must be a mount point
+   * @throws  FileDoesNotExistException if the provided {@code alluxioPath} does not exist
+   * @throws  InvalidPathException      if the provided {@code alluxioPath} is invalid and
+   *                                    information about the mount point cannot be found
+   * @throws  IOException               if an I/O-bound operation fails unexpectedly
+   * @throws  AccessControlException    if the parent lacks the necessary permissions
```
Comment (felipe-dna) [inline core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java:2825]: Throws must start with uppercase. Missing final point too.

File: core/common/src/main/java/alluxio/util/SecurityUtils.java [hunk 16]
```diff
@@ -139,7 +145,7 @@ public static String getGroup(UserState userState, AlluxioConfiguration conf) {
    * @param configuration   the {@link javax.security.auth.login.Configuration} to use
    * @param callbackHandler the {@link CallbackHandler}
    * @return                the new {@link LoginContext} instance
-   * @throws LoginException if LoginContext cannot be created for any reason whatsoever
+   * @throws LoginException if {@code LoginContext} cannot be created for any reason whatsoever
```
Comment (felipe-dna) [inline core/common/src/main/java/alluxio/util/SecurityUtils.java:148]: Throws must start with uppercase. Missing final point too.

File: core/common/src/main/java/alluxio/util/ConfigurationUtils.java [hunk 17]
```diff
@@ -537,13 +525,15 @@ public static GetConfigurationPResponse loadConfiguration(InetSocketAddress addr
...
+   * <p>
    * The given scope should only be {@link Scope#WORKER} or {@link Scope#CLIENT}.
    *
-   * @param properties the property list returned by grpc
-   * @param scope the scope to filter the received property list
-   * @param logMessage a function with key and value as parameter and returns debug log message
-   * @return the loaded properties
+   * @param properties  the property list returned by gRPC
+   * @param scope       the scope to filter the received property list, which should only be
+   *                    {@link Scope#WORKER} or {@link Scope#CLIENT}.
```
Comment (felipe-dna) [inline core/common/src/main/java/alluxio/util/ConfigurationUtils.java:534]: Remove this final point

File: core/common/src/main/java/alluxio/retry/ExponentialTimeBoundedRetry.java [hunk 18]
```diff
@@ -97,17 +97,25 @@ public Builder withTimeCtx(TimeContext timeCtx) {
...
 
     /**
-     * @param maxDuration max total duration to retry for
-     * @return the builder
+     * Sets the maximum duration for this retry policy.
+     * <p>
+     * Sets {@link #mMaxDuration} to retry for.
+     *
+     * @param   maxDuration max total duration to retry for
+     *                      in this {@link RetryPolicy}.
```
Comment (felipe-dna) [inline core/common/src/main/java/alluxio/retry/ExponentialTimeBoundedRetry.java:105]: Remove this final point

File: core/common/src/main/java/alluxio/Client.java [hunk 19]
```diff
@@ -40,6 +40,7 @@
   InetSocketAddress getAddress() throws UnavailableException;
 
   /**
+   * Gets the address from which to load the configuration for this client.
    * @return the {@link InetSocketAddress} of the configuration remote
    * @throws UnavailableException if the primary address cannot be determined
```
Comment (felipe-dna) [inline core/common/src/main/java/alluxio/Client.java:45]: Throws must start with uppercase. Missing final point too.

File: core/common/src/main/java/alluxio/AbstractClient.java [hunk 20]
```diff
@@ -456,6 +453,34 @@ public synchronized InetSocketAddress getConfAddress() throws UnavailableExcepti
...
+   * the established maximum number of retries.
+   *
+   * @param   rpc     the RPC call to be executed
+   * @param   onRetry the action to take on a retry
+   * @param   <V>     the type of the task where the RPC happens
+   * @return  the RPC result; otherwise, an exception is thrown
+   * @throws  FailedPreconditionException if the client is closed
+   * @throws  AlluxioStatusException      if a {@link StatusRuntimeException}
+   *                                      is thrown
+   * @throws  UnavailableException        if the RPC connection fails multiple times
```
Comment (felipe-dna) [inline core/common/src/main/java/alluxio/AbstractClient.java:480]: Throws must start with uppercase. Missing final point too.

File: core/common/src/main/java/alluxio/AbstractClient.java [hunk 21]
```diff
@@ -456,6 +453,34 @@ public synchronized InetSocketAddress getConfAddress() throws UnavailableExcepti
...
+   * <p>
+   * Throws an exception if the return of {@link RetryPolicy#getAttemptCount} is greater than
+   * the established maximum number of retries.
+   *
+   * @param   rpc     the RPC call to be executed
+   * @param   onRetry the action to take on a retry
+   * @param   <V>     the type of the task where the RPC happens
+   * @return  the RPC result; otherwise, an exception is thrown
+   * @throws  FailedPreconditionException if the client is closed
+   * @throws  AlluxioStatusException      if a {@link StatusRuntimeException}
```
Comment (felipe-dna) [inline core/common/src/main/java/alluxio/AbstractClient.java:478]: Throws must start with uppercase. Missing final point too.

File: core/common/src/main/java/alluxio/AbstractClient.java [hunk 22]
```diff
@@ -151,9 +151,19 @@ protected long getRemoteServiceVersion() throws AlluxioStatusException {
...
+   * Throws an exception if it is not. Does nothing otherwise.
+   * <p>
+   * Gets remote service version if {@link #mServiceVersion} is an
+   * {@link Constants#UNKNOWN_SERVICE_VERSION} and verifies if that
+   * corresponds to the provided {@code clientVersion}. Throws an
+   * exception if the versions diverge.
    *
-   * @param clientVersion the client version
+   * @param   clientVersion the client version
+   * @throws  IOException   if the {@link #mServiceVersion} is not the same as the
```
Comment (felipe-dna) [inline core/common/src/main/java/alluxio/AbstractClient.java:165]: Throws must start with uppercase. Missing final point too.

Comment (abnerferreira0212) [reply core/common/src/main/java/alluxio/AbstractClient.java:165]: > Throws must start with uppercase. Missing final point too.

Fixed.

File: core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java (hunk 14 above)
Comment (abnerferreira0212) [reply core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java:2877]: > Throws must start with uppercase. Missing final point too.

Fixed.

File: core/server/master/src/main/java/alluxio/master/file/FileSystemMaster.java (hunk 13 above)
Comment (abnerferreira0212) [reply core/server/master/src/main/java/alluxio/master/file/FileSystemMaster.java:420]: > Throws must start with uppercase. Missing final point too.

Fixed.

File: shell/src/main/java/alluxio/cli/fs/command/UnmountCommand.java (hunk 12 above)
Comment (abnerferreira0212) [reply shell/src/main/java/alluxio/cli/fs/command/UnmountCommand.java:70]: > Throws must start with uppercase. Missing final point too.

Fixed.

File: core/common/src/main/java/alluxio/AbstractClient.java (hunk 21 above)
Comment (abnerferreira0212) [reply core/common/src/main/java/alluxio/AbstractClient.java:478]: > Throws must start with uppercase. Missing final point too.

Fixed.

File: core/common/src/main/java/alluxio/AbstractClient.java (hunk 20 above)
Comment (abnerferreira0212) [reply core/common/src/main/java/alluxio/AbstractClient.java:480]: > Throws must start with uppercase. Missing final point too.

Fixed.

File: core/common/src/main/java/alluxio/Client.java (hunk 19 above)
Comment (abnerferreira0212) [reply core/common/src/main/java/alluxio/Client.java:45]: > Throws must start with uppercase. Missing final point too.

Fixed.

File: core/common/src/main/java/alluxio/retry/ExponentialTimeBoundedRetry.java (hunk 18 above)
Comment (abnerferreira0212) [reply core/common/src/main/java/alluxio/retry/ExponentialTimeBoundedRetry.java:105]: > Remove this final point

Fixed.

File: core/common/src/main/java/alluxio/security/login/AlluxioLoginModule.java (hunk 1 above)
Comment (abnerferreira0212) [reply core/common/src/main/java/alluxio/security/login/AlluxioLoginModule.java:71]: > The final point must be after the (), what you think?

Fixed.

File: core/common/src/main/java/alluxio/security/user/UserStateFactory.java (hunk 2 above)
Comment (abnerferreira0212) [reply core/common/src/main/java/alluxio/security/user/UserStateFactory.java:23]: > Missing final point

Fixed.

File: core/common/src/main/java/alluxio/util/ConfigurationUtils.java (hunk 17 above)
Comment (abnerferreira0212) [reply core/common/src/main/java/alluxio/util/ConfigurationUtils.java:534]: > Remove this final point

Fixed.

File: core/common/src/main/java/alluxio/util/SecurityUtils.java (hunk 16 above)
Comment (abnerferreira0212) [reply core/common/src/main/java/alluxio/util/SecurityUtils.java:148]: > Throws must start with uppercase. Missing final point too.

Fixed.

File: core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java (hunk 15 above)
Comment (abnerferreira0212) [reply core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java:2825]: > Throws must start with uppercase. Missing final point too.

Fixed.
